File: 1-INICIANTE/1010/1010.py
peca1 = input()
peca2 = input()

peca1 = peca1.split()
peca2 = peca2.split()


total_peca1 = int(peca1[1]) * float(peca1[2])
total_peca2 = int(peca2[1]) * float(peca2[2])

total_pagar = total_peca1 + total_peca2

print(f"VALOR A PAGAR: R$ {total_pagar:.2f}")

File: 1-INICIANTE/1010/1010.go
package main

import "fmt"

func main() {
	var codigo1, codigo2, quantidade1, quantidade2 int
	var preco1, preco2 float64

	fmt.Scan(&codigo1, &quantidade1, &preco1)
	fmt.Scan(&codigo2, &quantidade2, &preco2)

	totalpeca1 := float64(quantidade1) * preco1
	totalpeca2 := float64(quantidade2) * preco2

	totalPagar := totalpeca1 + totalpeca2

	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", totalPagar)
}
